src/puppetlabs/trapperkeeper/services_internal.clj

```diff
@@ -139,6 +139,7 @@
   (let [f (first forms)]
     (cond
       (symbol? f) (merge {:service-protocol-sym f} (validate-deps-form! (rest forms)))
+      (map? f) (merge {:service-protocol-sym nil} (validate-deps-form! forms))
```

Comment:
Nah, I think that when we port this fn to schema as you described in your comment, we'll be able to take care of that then.  So I'm good with this for now.